products/zoningtaxlots/models/intermediate/_int_models.yml

```diff
@@ -2,12 +2,12 @@ version: 2
 
 models:
   - name: int__commercialoverlay
-    tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - dtm_id
-            - overlay
-            - row_number
+    columns:
```

Comment:
And with the tweak I'm suggesting in commercial overlay, this should go back to unique combo of dtm_id and overlay

Reply:
@fvankrieken It didn't pass this test, maybe I have to either deduplicate before adding row numbers or keep the deduplicate CTEs.

Reply:
Gotcha. Let's do a group by before the ordering in that case (grouping by `dtm_id, overlay`)

Reply:
And unioning/summing areas in the group by (similar to how we do for both special districts and zoning districts)